packages/@glimmer/runtime/lib/references.ts

```diff
@@ -62,20 +63,13 @@ export const TRUE_REFERENCE: PrimitiveReference<boolean> = new ValueReference(tr
 export const FALSE_REFERENCE: PrimitiveReference<boolean> = new ValueReference(false);
 
 export class ConditionalReference extends CachedReference<boolean> implements Reference<boolean> {
-  constructor(
-    private inner: Reference<unknown>,
-    private toBool: (value: unknown) => boolean = defaultToBool
-  ) {
+  constructor(private inner: Reference<unknown>, private toBool = defaultToBool) {
     super();
   }
```

Comment:
Thinking:

```suggestion
  constructor(private inner: Reference<unknown>) {
    super();
    
    this.toBool = toBool;
  }
```

Then the custom subclass can customize in its constructor after calling super...